src/app/containers/Preferences/General/Aaxis.js

```diff
@@ -0,0 +1,59 @@
+import controller from 'app/lib/controller';
+import React, { useState } from 'react';
+import store from 'app/store';
+import ToggleSwitch from 'app/components/ToggleSwitch';
+import Tooltip from 'app/components/TooltipCustom/ToolTip';
+import Input from '../components/Input';
+import Fieldset from '../components/Fieldset';
+
+
+const Aaxis = () => {
+    const [addAAxisStepPerMM, setAddAAxisStepPerMM] = useState(false);
+    //TODO
+    const handleRotaryAxisMMToggle = (addAAxisStepPerMM) => {
+        setAddAAxisStepPerMM(!addAAxisStepPerMM);
+        // const diameter = 1; //TODO
+        //Max y movement = circumference(pi * d)
+        // const ymax = 3.14 * diameter;
+        if (addAAxisStepPerMM) {
+            //updatedYresolution or $101 = 9600/[ymax]
+            // $101 = 9600/ymax;
+            // //Y-axis Steps per mm adjusted for project
+            // M0
+            // G0 Y[ymax]
+            // G4 P500
+            // G0 Y0
+
+            const mmMacro = ''; //macro content
+
+            //Run macro
+            controller.command('macro:rotaryMMenable', mmMacro);
+            return;
+        }
+        controller.command('macro:rotaryMMdisable');
+    };
```

Comment:
- Would prefer better naming conventions for the functions and variables, don't need to specify the MM. Ex. the function name "handleRotaryAxisMMToggle" can just be "**handleRotaryAxisToggle**"
- Comments here can just be removed, don't have to worry about that logic now, it could possibly not even on the front-end